public_site/src/site/resources/util/aem-binary-artifact-name-generator.html

```diff
@@ -117,7 +117,7 @@
 
         var aemSdkRegex = /^aem-sdk-(.+)\.zip$/;
         var aemSdkQuickstartRegex = /^aem-sdk-quickstart-(.+)\.jar$/;
-        var aemSdkAddonRegex = /^aem-.+-addon-(.+)\.zip$/;
+        var aemSdkAddonRegex = /^(aem|commerce)(-.+)*-addon-(.+)\.zip$$/;
```

Comment:
there is a double $$ at the end - is this by intention?

Reply:
No, of course not - Sorry, I've fixed it